youtube-gpt-client/src/components/header/index.tsx

```diff
@@ -1,20 +1,29 @@
-import { Flex, Heading, Link } from '@chakra-ui/react';
+import { Flex, Heading } from '@chakra-ui/react';
+import { useTranslation } from 'react-i18next';
+import LanguageSwitcher from '@/components/header/languageSwitcher';
+import Link from 'next/link';
 
 const Header = () => {
+  const { t, i18n } = useTranslation('menu');
   return (
     <Flex
       justifyContent="space-between"
       alignItems="center"
       padding="15px 50px"
       color="#fff"
-      height="50px"
+      height="60px"
     >
       <Heading size="md">
         <Link href="/">Youtube GPT</Link>
       </Heading>
-      <Flex gap="20px">
-        <Link href="/how-it-works">How it works</Link>
-        <Link href="/team">Team</Link>
+      <Flex gap="20px" fontSize="20px">
+        <Link href="/how-it-works" locale={i18n.language}>
```

Comment:
is this locale={i18n.language} needed? I see it repeated and wondering if it can be solved in some global config instead of in every use?

Reply:
I've updated the code to store the [NEXT_LOCALE cookie] (https://nextjs.org/docs/pages/building-your-application/routing/internationalization#leveraging-the-next_locale-cookie). This way, we don't need to set all links and work in the entire application. I'm using [nookies](https://github.com/maticzav/nookies) to set the value because we can retrieve the cookies from the server side and client side of the application with ease.